### PR TITLE
Docs: Deployment section in our 'User Guides'

### PR DESCRIPTION
**Pull request by dcfidalgo**
This PR adds a `Deployment` section to the *User Guides* of the documentation. Goal is to describe two deployment options:
- built-in tool via our CLI and FastAPI
- deployment via MLFlow models

There is still little content and It's still a draft, @ignacioct maybe we could work together on this branch.
@dvsrepo in case you want to follow the progress and comment: checking out this branch and running `make docs` should provide you a preview of the docs.

### CODE REVIEW

File: docs/docs/documentation/user-guides/3-deployment.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Deployment
+
+*Biome.text* provides an easy to use built-in tool to deploy your model on a local machine as a REST API via [FastAPI](https://fastapi.tiangolo.com/).
+Additionally, you can easily export your pipeline to an [MLFlow Model](https://mlflow.org/docs/latest/models.html#mlflow-models)
+and take advantage of all its deployment tools like packaging the model as self-contained Docker image with a REST API endpoint,
+or deploying it directly on Microsoft Azure ML or Amazon SageMaker.
```
Comment (ignacioct) [inline docs/docs/documentation/user-guides/3-deployment.md:6]:
```suggestion
and take advantage of all its deployment tools, like packaging the model as self-contained Docker image with a REST API endpoint or deploying it directly on Microsoft Azure ML or Amazon SageMaker.
```
super small gramatical detail

**Review by ignacioct — COMMENTED** [review]
Super small gramatical change suggested. 

And yeah, I can help documenting this part, and it also goes over what I've got to do in my thesis, so it's ideal.

**Review by ignacioct — APPROVED** [review]
LGTM!
There are a few not implemented methods, but I understand this is a work in progress. 
The new documentation guide looks quick and very understandable for me :)

File: docs/docs/documentation/user-guides/3-deployment.md (hunk 1 above)
Comment (dcfidalgo) [reply docs/docs/documentation/user-guides/3-deployment.md]: Thanks, fixed!

**Pull request merged**